test/e2e/scalability/load.go

```diff
@@ -703,3 +720,30 @@ func CreateNamespaces(f *framework.Framework, namespaceCount int, namePrefix str
 	}
 	return namespaces, nil
 }
+
+func CreateQuotas(f *framework.Framework, namespaces []*v1.Namespace, podCount int, testPhase *timer.Phase) error {
+	defer testPhase.End()
+	quotaTemplate := &v1.ResourceQuota{
+		Spec: v1.ResourceQuotaSpec{
+			Hard: v1.ResourceList{"pods": *resource.NewQuantity(int64(podCount), resource.DecimalSI)},
+		},
+	}
+
+	for _, ns := range namespaces {
+		var got *v1.ResourceQuota
+		if err := wait.PollImmediate(2*time.Second, 30*time.Second, func() (bool, error) {
+			var err error
```

Comment:
I guess you don't need to define here. You can just directly define it inside the 'if' command.

Reply:
Done.